rolling

```diff
@@ -1 +1 @@
-focal
+groovy
```

Comment:
This shouldn't be updated until `groovy` is considered released, right?

cc @mwhudson @philroche

Reply:
I agree

Reply:
Right. Thank you for catching this.